libs/json/encoder.go

```diff
@@ -28,7 +28,7 @@ func Marshal(v interface{}) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-// MarshalIndent marshals the value as JSON, using the given prefix and indentation.
+// MarshalIndent marshals the value as JSON, using the given prefix and indention.
```

Comment:
```suggestion
// MarshalIndent marshals the value as JSON, using the given prefix and indentation.
```

Incorrect detection of a spelling error.